kuma/javascript/src/newsletter.jsx

```diff
@@ -145,6 +145,9 @@ export default function Newsletter() {
         if (disableNewsletter) {
             setShowNewsletter(false);
         }
+        // if JS is enabled client-side, hide the privacy checkbox until
+        // the email input field receive focus
+        setShowPrivacyCheckbox(false);
```

Comment:
Beyond the checkbox, does signing up work at all with JS disabled?

Reply:
Technically yes, as it will post to `https://www.mozilla.org/en-US/newsletter/`